subworkflows/local/allele_specific_calling.nf

```diff
@@ -31,9 +32,16 @@ workflow ALLELE_SPECIFIC_CALLING {
     main:
         ch_versions = Channel.empty()
 
+        // Keep only one variant per position in the vcf
+        BCFTOOLS_NORM(
+            ch_ind_vcf_tbi,
+            ch_fasta
+        )
+        TABIX_NORM(BCFTOOLS_NORM.out.vcf)
```

Comment:
If you update bcftools to 1.20, you could now use `--write-index` instead of `TABIX_TABIX` 🤓

Reply:
That´s super nice! I didn't know that 😮

Reply:
Yes, there are downstream implications but GATK4_ASEREADCOUNTER will fail if there are two variants present in the same position, and as far as I could see, that couldn't be done with bcftools view. The reason to perform the bcftools norm here instead of earlier within the call_variant subworkflow is to just affect GATK4_ASEREADCOUNTER, the output of which will be used to annotate further the original vcf (where the variants with duplicated positions have not been filtered out)

Reply:
Ah, I misread and thought it was the GATK4_ASEREADCOUNTER that went further. Then it seems more alright.